robotpy_ext/control/toggle.py

```diff
@@ -65,14 +66,15 @@ def get(self):
             else:
                 return False
 
-    def __init__(self, joystick, button, debouncePeriod=None):
+    def __init__(self, joystick: wpilib.Joystick, button: int, debounce_period=None):
         """
-        :param joystick: wpilib.Joystick that contains the button to toggle
-        :param button: Value of button that will act as toggle. Same value used in getRawButton()
+        :param joystick: :class:`wpilib.Joystick` that contains the button to toggle
+        :param button: Number of button that will act as toggle. Same value used in `getRawButton()`
+        :param debounce_period: Period to wait before registering a new button press.
         """
 
-        if debouncePeriod is not None:
-            self.joystick = Toggle._SteadyDebounce(joystick, button, debouncePeriod)
+        if debounce_period is not None:
+            self.joystick = Toggle._SteadyDebounce(joystick, button, debounce_period)
         else:
             self.joystick = joystick
             self.joystick.get = partial(self.joystick.getRawButton, button)
```

Comment:
Uh, this mutates the joystick you get. Bad idea. Won't be able to use this twice on the same joystick if you don't specify `debounce_period`.

Reply:
Hmm you're right...